src/app/check/steps.js

```diff
@@ -27,9 +27,14 @@ module.exports = {
         value: "retryNationalInsurance",
         next: "national-insurance-number",
       },
-      "/oauth2/callback",
+      "abandon",
     ],
   },
+  "/abandon": {
+    skip: true,
+    controller: AbandonController,
+    next: "/oauth2/callback",
```

Comment:
Do you think we could add a browser test for this?

Reply:
Hmm yeah don't think that's possible, but if a user has successfully abandoned[ "they should redirected as access denied"](https://github.com/govuk-one-login/ipv-cri-check-hmrc-front/blob/0a63eff860d4e927dcd9abb3c693aa9a6c812121/tests/browser/features/abandon-nino-cri.feature#L26) as per https://govukverify.atlassian.net/wiki/spaces/Architecture/pages/3709207192/Request+Response+formats#Error-handling

Reply:
Ok I have updated this now so it does check the redirect is access_denied.